gpkit/__init__.py

```diff
@@ -90,11 +90,8 @@ def disable_signomials():
 from .variables import Variable, VectorVariable, ArrayVariable
 from .varkey import VarKey
 from .posyarray import PosyArray
-from .geometric_program import GeometricProgram
-from .signomial_program import SignomialProgram
```

Comment:
Why remove `GeometricProgram` and `SignomialProgram` but keep `GP` and `SP`? Isn't that the point of `gpkit.shortcuts`? I think that if we provide `GP` we should also provide the full class name `GeometricProgram`.